pkg/tmpl/context_funcs.go

```diff
@@ -69,6 +70,14 @@ func (c *Context) createFuncMap() template.FuncMap {
 		funcMap["readFile"] = func(string) (string, error) {
 			return "", nil
 		}
+		funcMap["readDir"] = func(string) ([]string, error) {
+			var paths []string
+			return paths, nil
+		}
+		funcMap["listDir"] = func(string) ([]string, error) {
+			var paths []string
```

Comment:
return nil,nil

Reply:
@yxxhero Are you sure, readFile function here returns empty string not nil?

Reply:
empty slice can be nil, but empty string can't

Reply:
so maybe we should go with paths []string{}  instead of paths []string?

Reply:
return []string{}, nil?

Reply:
@yxxhero sure, then it will be empty array as I understand

Reply:
@vasicvuk It's ok too.

Reply:
Thanks :), I will modify it a little later when I am near the PC